charts/kyverno/README.md

```diff
@@ -92,6 +92,9 @@ The following table lists the configurable parameters of the kyverno chart and t
 | `podAnnotations`                   | annotations to add to each pod                                                                                                                                                                                                                           | `{}`                                                                                                                                                                                                                                                                     |
 | `podLabels`                        | additional labels to add to each pod                                                                                                                                                                                                                     | `{}`                                                                                                                                                                                                                                                                     |
 | `podSecurityContext`               | security context for the pod                                                                                                                                                                                                                             | `{}`                                                                                                                                                                                                                                                                     |
+| `podDisruptionBudget.enabled`      | Adds a PodDisruptionBudget for the kyverno deployment                                                                                                                                                                                                    | `false`                                                                                                                                                                                                                                                                  |
+| `podDisruptionBudget.minAvailable` | Configures the minimum available pods for kyverno disruptions. Cannot used if `maxUnavailable` is set.                                                                                                                                                   | `1`                                                                                                                                                                                                                                                                      |
```

Comment:
Is the last para the default value? Shouldn't it be `0`?

Reply:
Yes, updated.